median/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"sort"
+)
+
+func median(intsArr []int) float64 {
+	var answer float64
+	val := len(intsArr) / 2
+	log.Println("Half value: ", val)
+	if len(intsArr)%2 == 1 {
+		log.Println("%2 == 1")
+		answer = float64(intsArr[val])
+		log.Println("Answer:", answer)
+	} else {
+		log.Println("%2 == 0")
+		answer = (float64(intsArr[val-1]) + float64(intsArr[val])) / 2
+		log.Println("Answer:", answer)
+	}
+	return answer
+}
+
+func main() {
+	intsArr := []int{7, 2, 4, 8, 1, 2, 9}
+	// intsArr := []int{7, 2, 4, 8, 1, 9}
+	sort.Ints(intsArr)
```

Comment:
If I added some `file.go` to the package and wanted to use `median()` func, I could miss the fact that the slice (`intsArr []int` - `median()` parameter) must be sorted. Please move you sorting inside `median()` func.